README.md

```
@@ -5,6 +5,14 @@ Fast and cheap sql builder.
 [![Go Reference](https://pkg.go.dev/badge/github.com/profe-ajedrez/obreron/v2.svg)](https://pkg.go.dev/github.com/profe-ajedrez/obreron/v2)
 [![Go Report Card](https://goreportcard.com/badge/github.com/profe-ajedrez/obreron/v2)](https://goreportcard.com/report/github.com/profe-ajedrez/obreron/v2)
 
+# Obreron
```

Markdownlint

Multiple headings with the same content

(MD024, no-duplicate-heading)